### ISSUE TITLE
Background images on training

### ISSUE THREAD

**Issue by aseprohman** [ultralytics/yolov5#5231]
## ❔Question
Hello @glenn-jocher 
in yolov4 alexeyab adding background images to training dataset is by adding image with empty label txt file, in your framework apps "no label is required" for background images, you mean I don't need to create empty txt files ? 
and where should I put the background image ? train / validation set ?

**Comment by glenn-jocher** [comment]
@aseprohman you don't need any labels for background images in YOLOv5, you just add the images to your dataset.

**Comment by aseprohman** [comment]
many thanks :)
in one or both of ( train / val ) set for best model performance result ?

**Comment by ChinaRush** [comment]
both of train/val set need background images for best result

**Comment by aseprohman** [comment]
nice info, many thx :)

**Comment by anhtt20172948** [comment]
Do I need to add __background__ class in config file as cocodataset to use background training ? Because I am using my dataset and currently there is no label __background__

**Comment by glenn-jocher** [comment]
@uiui1999vn no. To use background images simply add images to your dataset with no labels.
